service/frontend/workflowHandler.go

```diff
@@ -342,7 +342,7 @@ func (wh *WorkflowHandler) StartWorkflowExecution(ctx context.Context, request *
 		return nil, wh.error(errInvalidExecutionStartToCloseTimeoutSeconds, scope)
 	}
 
-	if request.GetTaskStartToCloseTimeoutSeconds() <= 0 {
+	if request.GetTaskStartToCloseTimeoutSeconds() < 0 {
```

Comment:
Why is this? Does `0` mean: don't wait a single nanosecond?

Reply:
0 mean "use default"

Reply:
Or we need to switch all of them to some nullable type

Reply:
I would leave it as `0`.